.circleci/config.yml

```diff
@@ -82,7 +82,7 @@ commands:
     parameters:
       redis_version:
         type: string
-        default: "7.2"
```

Comment:
why do we need to change it in circleci?

Reply:
I think we should delete it, I agree. As a separate commit should be fine.